lib/MusicBrainz/Server/Edit/Medium/Create.pm

```diff
@@ -175,26 +176,6 @@ override 'to_hash' => sub
     return $hash;
 };
 
-sub allow_auto_edit {
```

Comment:
This was already implemented for `$EDIT_MEDIUM_CREATE`, so I basically just moved the code it has to a role.